templates/python/custom-copilot-basic/src/bot.py.tpl

```diff
@@ -0,0 +1,68 @@
+import os
+import sys
+import traceback
+
+from botbuilder.core import MemoryStorage, TurnContext
+from teams import Application, ApplicationOptions, TeamsAdapter
+from teams.ai import AIOptions
+from teams.ai.models import AzureOpenAIModelOptions, OpenAIModel, OpenAIModelOptions
+from teams.ai.planners import ActionPlanner, ActionPlannerOptions
+from teams.ai.prompts import PromptManager, PromptManagerOptions
+from teams.state import TurnState
+
+from config import Config
+
+config = Config()
+
+# Create AI components
+model: OpenAIModel
+
+{{#useAzureOpenAI}}
+model = OpenAIModel(
+    AzureOpenAIModelOptions(
+        api_key=config.AZURE_OPENAI_API_KEY,
+        default_model=config.AZURE_OPENAI_MODEL_DEPLOYMENT_NAME,
+        endpoint=config.AZURE_OPENAI_ENDPOINT,
+    )
+)
+{{/useAzureOpenAI}}    
+{{#useOpenAI}}
+model = OpenAIModel(
+    OpenAIModelOptions(
+        api_key=config.OPENAI_API_KEY,
+        default_model=config.OPENAI_MODEL_DEPLOYMENT_NAME,
```

Comment:
@kuojianlu @MuyangAmigo @blackchoey  Please review if it is comfortable for you about deployment name here.